### PR TITLE
docs: cleanup/polish Markdown files

### PR DESCRIPTION
**Pull request by Nerixyz**
These are some minor, (mostly) stylistic changes/fixes to the markdown files.

- Fixed some language ✨stuff✨.
- Added alternative to httpbin.
- Updated expected space requirement on Windows.
- Removed unused VS component on Windows.
- Moved Qt Creator formatting to Windows docs.
- Updated nativs link to Qt 6.
- Added missing language to code blocks.
- Removed `# Description` from PR template and added instructions to fix a GitHub issue.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-To run tests all tests you will need to run the `kennethreitz/httpbin` and `ghcr.io/chatterino/twitch-pubsub-server-test:latest` docker images
+To run tests all tests you will need to run the `kennethreitz/httpbin` and `ghcr.io/chatterino/twitch-pubsub-server-test:latest` docker images.
```
Comment (Wissididom) [inline tests/README.md:1]:
```suggestion
To run all tests you will need to run the `kennethreitz/httpbin` and `ghcr.io/chatterino/twitch-pubsub-server-test:latest` docker images.
```

File: README.md [hunk 2]
```diff
@@ -58,19 +53,9 @@ file](./.git-blame-ignore-revs). GitHub does this by default.
...
-### Get it automated with QT Creator + Beautifier + Clang Format
-
-1. Download LLVM: https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.6/LLVM-16.0.6-win64.exe
-2. During the installation, make sure to add it to your path
-3. In QT Creator, select `Help` > `About Plugins` > `C++` > `Beautifier` to enable the plugin
-4. Restart QT Creator
-5. Select `Tools` > `Options` > `Beautifier`
-6. Under `General` select `Tool: ClangFormat` and enable `Automatic Formatting on File Save`
-7. Under `Clang Format` select `Use predefined style: File` and `Fallback style: None`
+The code is formatted using [ClangFormat](https://clang.llvm.org/docs/ClangFormat.html) in Qt Creator. [.clang-format](.clang-format) contains the style file for ClangFormat.
```
Comment (pajlada) [inline README.md:56]:
```suggestion
The code is formatted using [clang-format](https://clang.llvm.org/docs/ClangFormat.html). Our configuration is found in the [.clang-format](.clang-format) file in the repository root directory.
```

File: tests/README.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
 
 For example:
 
 ```bash
 docker run --network=host --detach ghcr.io/chatterino/twitch-pubsub-server-test:latest
 docker run -p 9051:80 --detach kennethreitz/httpbin
 ```
+
+Alternatively, you can use [httpbox](github.com/kevinastone/httpbox) (`httpbox --port 9051`).
```
Comment (pajlada) [inline tests/README.md:10]: Phrase this as: "If you're unable to use docker..." & include a link to https://github.com/Chatterino/twitch-pubsub-server-test/releases/latest too

File: BUILDING_ON_WINDOWS.md [hunk 4]
```diff
@@ -120,57 +122,69 @@ Then in a terminal, configure conan to use `NMake Makefiles` as its generator:
...
 - Click the hammer on the bottom left to generate a build (does not run the build though).
 
 Build results will be placed in a folder at the same level as the "chatterino2" project folder (e.g. if your sources are at `C:\Users\example\src\chatterino2`, then the build will be placed in an automatically generated folder under `C:\Users\example\src`, e.g. `C:\Users\example\src\build-chatterino-Desktop_Qt_6.5.3_MSVC2019_64bit-Release`.)
 
-- Note that if you are building chatterino purely for usage, not for development, it is recommended that you click the "PC" icon above the play icon and select "Release" instead of "Debug".
+- Note that if you are building Chatterino purely for usage, not for development, it is recommended that you click the "PC" icon above the play icon and select "Release" instead of "Debug".
 - Output and error messages produced by the compiler can be seen under the "4 Compile Output" tab in Qt Creator.
 
-#### Producing standalone builds
+#### Producing Standalone Builds
```
Comment (pajlada) [inline BUILDING_ON_WINDOWS.md:159]: Revert the "titlecaseification" of this & other headers - the rewording of them all is fine though

**Review by pajlada — COMMENTED** [review]
Some nits, overall looks good

**Review by pajlada — APPROVED** [review]

**Pull request merged**